app/src/main/java/com/teamagam/gimelgimel/app/common/logging/DiskLogger.java

```diff
@@ -198,7 +198,7 @@ private void logToFile(String verbosity, String message) {
         try {
             mLogWriter.write(completeLogLine);
         } catch (IOException e) {
-            throw new LogWriteException();
```

Comment:
That doesn't seems like the solution.....
At least use the native logger to log it, if not understand if there's anything else we can do (retry X times?) !fix